passgen/diceware/build.gradle.kts

```diff
@@ -4,10 +4,13 @@
  */
 
 plugins {
-  kotlin("jvm")
+  id("com.github.android-password-store.android-library")
+  id("com.github.android-password-store.kotlin-android")
```

Comment:
Any particular reason as to why we need it to be an android library? I don't think we're accessing any android-specific resource 🤔

Reply:
Loading resources from JVM libraries is very inefficient on Android and causes [excessive memory usage](https://blog.danlew.net/2013/08/20/joda_time_s_memory_issue_in_android/).

Reply:
Can we keep the wordlist in the app module?

Reply:
I'd prefer if it stays in the library module so that it can be verified by `WordListParserTest`.

Reply:
Cool, I was looking at it from a reusability perspective (for other people) but I guess anyone can easily clone the repo and remove the `android-library` plugin

Reply:
We're not publishing this library either way